src/sources/statsd/mod.rs

```diff
@@ -113,6 +119,17 @@ pub struct TcpConfig {
     /// The maximum number of TCP connections that are allowed at any given time.
     #[configurable(metadata(docs::type_unit = "connections"))]
     connection_limit: Option<u32>,
+
+    /// sanitize
+    ///	By default all stat names (keys) are sanitized.
+    ///	  "/" is replaced with "-"
+    ///	  All whitespace is replaced with "_"
+    ///	  All non alphanumeric characters [^a-zA-Z_\-0-9\.] are removed.
+    ///	For backwards compatibility this remains the default behavior.
+    ///	To disable sanitization, set to false
```

Comment:
```suggestion
    ///	Whether to sanitize incoming statsd key names. When `true`, keys are sanitized by:
    /// - "/" is replaced with "-"
    /// - All whitespace is replaced with "_"
    ///- All non alphanumeric characters [^a-zA-Z_\-0-9\.] are removed.
```

This doc comment ends up in the generated documentation so I cleaned this up a bit. The default will be shown in the docs, so it doesn't need to be part of the comment.